docs/wiki/The CI environment - Pipeline usage.md

```diff
@@ -0,0 +1,141 @@
+1. Push the local changes to the repository (using a branch that is not [main\|master]).
+1. On the DevOps platform, navigate to your pipelines and select the pipeline that was registered for the module you updated.
+1. Select the branch with your updated template.
+1. (Optionally) disable the `Remove deployed module` input parameter in case you don't want to apply the default behavior, and want to skip the deletion of the test-deployed resource to check it after deployment.
+1. (Optionally) adjust the `Publish prerelease module` flag in case you want to publish a prerelease version of your updated module from your development branch.
+1.  Trigger the pipeline.
+
+Once the pipeline concludes, it will either be in a green (success) or red (failed) state, depending on how the module performed.
+
+Pipeline logs are available for troubleshooting and provide detailed information in case of failures. If errors occur in the [Static validation](./The%20CI%20environment%20-%20Static%20validation) phase, you may only see the failed test and need to `expand` the error message. How this looks like depends on the [DevOps platform](#devops-tool-specific-guidance) you use.
+
+## Add a new module pipeline
+
+To add a new module pipeline we recommend to create a copy of a currently existing module pipeline and adjust all module-specific properties. The registration of the pipeline depends on the [DevOps platform](#devops-tool-specific-guidance) you're using.
+# Operate the dependency pipeline
+
+The dependency pipeline must be triggered manually and deploys a set of resources we reference as part of our module tests (e.g. Virtual Networks, Log Analytics Workspace, Key Vaults).
+
+Triggering the pipeline is as easy as navigating to it in your corresponding DevOps tool and running the pipeline.
+
+> **Note:** While operating the dependency pipeline is simple, make sure to set it up correctly as described in the [Getting Started - Dependency pipeline](./Getting%20started%20-%20Dependency%20pipeline) section.
+
+## Add a new dependency
+
+Depending on what you want to test in your module pipeline, you may want to include additional resources to your dependency pipeline. If so, make sure to include an additional parameter file for each service you require under `utilities/pipelines/dependencies`. Once done, you need to add the deployment to the pipeline itself making sure to deploy the new resources in the correct order. The implementation depends on the [DevOps platform](#devops-tool-specific-guidance) you're using.
+
+---
+
+# DevOps-Tool-specific guidance
+
+This section provides a step-by-step guideline on how to operate the pipelines based on the specific chosen DevOps platform, GitHub or Azure DevOps.
+
+## GitHub workflows
+
+This section focuses on _GitHub_ Actions & Workflows.
+
+<details>
+<summary>GitHub workflows</summary>
+
+  ### Trigger a workflow
+
+  To trigger a workflow in _GitHub_:
```

Comment:
As before: Should we add some guidance around the fact that it's not possible to just 'register' a new pipeline from a branch?